README.md

```diff
@@ -101,33 +96,33 @@ See the results in [h2oai's db-benchmark](https://h2oai.github.io/db-benchmark/)
 
 Install the latest polars version with:
 
+```sh
+pip install polars
 ```
-# Install Polars only.
-$ pip3 install -U 'polars'
 
-# Install Polars with all optional dependencies.
-$ pip3 install -U 'polars[all]'
-
-# Install Polars and numpy.
-$ pip3 install -U 'polars[numpy]'
-
-# Install Polars and pyarrow/pandas/numpy to be able to convert to/from pandas and/or read data with pyarrow.
-$ pip3 install -U 'polars[pyarrow]'
-
-# Install Polars and pyarrow/pandas/numpy and fsspec (read from e.g. remote filesystems, compressed files).
-$ pip3 install -U 'polars[pyarrow,fsspec]'
+or on Conda
```

Comment:
Can we add a note that pip is favored to get the latest version/patches.

Reply:
Done, I have conda install instruction less prominent and emphasized that pip is the preferred way.